components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -93,14 +94,20 @@ internal sealed class SubscriptionWrapper : ISubscription {
             /// <inheritdoc/>
             public ConnectionModel Connection => _subscription.Connection;
 
-            public ConcurrentDictionary<string, MonitoredItemNotificationModel> PendingAlarms { get; } = new ConcurrentDictionary<string, MonitoredItemNotificationModel>();
-
             /// <inheritdoc/>
             public event EventHandler<SubscriptionNotificationModel> OnSubscriptionDataChange;
 
             /// <inheritdoc/>
             public event EventHandler<SubscriptionNotificationModel> OnSubscriptionEventChange;
 
+            /// <inheritdoc/>
```

Comment:
There's no summary to inherit here. Can you please add it for these event handlers?

Reply:
Yes, there is in the ISubscription class

Reply:
You are right. Sorry!